<?xml version="1.0"?>
<launch>
    <arg name="common_modbus_ip"      default="192.168.1.111" />
    <node pkg="camel_robot" name="modbus" type="modbus.py" output="screen" ns="common" required='true'>
        <param name="ip_address" value="$(arg common_modbus_ip)"/>
        <param name="port" value="502"/>
        <param name="rate" value="10"/>
    </node> 
    <node pkg="camel_robot" name="hook" type="hook_cntrl.py" output="screen"  required='true' ns="common"/>
    <!-- <node pkg="camel_robot" name="test" type="test_hook.py" output="screen"  required='true' ns="common"/> -->
    <node pkg="camel_robot" name="light_control" type="Light_control.py" output="screen"  required='true' ns="common"/>
    <node pkg="camel_robot" name="light_service" type="Light_service.py" output="screen"  required='true' ns="common"/>
</launch>
